src/components/Header/Header.tsx

```diff
@@ -0,0 +1,76 @@
+'use client'
+
+import { BottomNavigation, BottomNavigationAction, Box } from '@mui/material'
+import React from 'react'
+
+interface Props {}
+
+const style = {
+  link: {
+    fontSize: 14,
+    fontWeight: '600',
+    cursor: 'pointer',
+    borderRadius: '6px',
+    paddingX: '0.8rem',
+    paddingY: '0.4rem',
+    '&:hover': {
+      transform: 'translateY(0.4rem)',
+      backgroundColor: 'black',
+      color: 'white',
+    },
+    '&:active': {
+      transform: 'scale(0.90) translateY(0.4rem)',
+    },
+    '& .Mui-selected': {
+      color: '#2563eb',
+    },
+    transition: 'all 240ms cubic-bezier(0.4, 0.4, 0.4, 0.4)',
+  },
+}
+
+function Header({}: Props) {
+  const [value, setValue] = React.useState(0)
+  return (
+    <Box
+      component={'header'}
+      sx={{
+        width: '100vw',
+        display: 'flex',
+        justifyContent: 'center',
+        borderBottom: 2,
+        paddingY: '1.2rem',
+        borderColor: '#e7e5e4',
+      }}
+    >
+      <Box sx={{ display: 'flex', flexDirection: 'column', rowGap: '0.4rem' }}>
+        <Box
+          sx={{
+            fontSize: 28,
+            fontWeight: '800',
+          }}
+        >
+          OpenBC
+        </Box>
+
+        <BottomNavigation
+          showLabels
+          value={value}
+          onChange={(event, newValue) => {
+            if (typeof newValue === 'number') setValue(newValue)
```

Comment:
Curious what necessitated the `if` statement here?

Doesn't seem to be something that shows up in the `BottomNavigation` component docs:

https://mui.com/material-ui/react-bottom-navigation/#bottom-navigation

Reply:
It is a type check to make sure it is a number. Have a look at line 10, const [value, setValue] = React.useState(0), I just want to make sure that the new value pass in setValue is a number.

Reply:
We are using the typescript-eslint. It is very strict on the types. Here is the reference: [https://typescript-eslint.io/rules/no-unsafe-argument/](url). And here is the error if I remove the if:
![image](https://github.com/OpenBCca/openbc-web/assets/72838071/0f9f8066-3220-441b-83ac-d42484d69cde)

Reply:
Ahh, I see, in that case, I think the more Typescript-y way of doing this would be:

https://github.com/OpenBCca/openbc-web/pull/4/files#r1385906362

Reply:
@CorreyL , I think the issue with your suggestion is that the onChange value is any.

Reply:
I suggest changing it to the following

It's more clear and returns early on unexpected value;

`
if (typeof newValue !== 'number') {
    return;
}

 setValue(newValue)
`